package.json

```diff
@@ -5,16 +5,15 @@
   "homepage": "https://replicache.dev",
   "repository": "github:rocicorp/replicache-react",
   "type": "module",
-  "main": "out/index.js",
-  "module": "out/index.mjs",
+  "module": "out/index.js",
```

Comment:
"module" is still needed to make Parcel happy.
https://parceljs.org/blog/v2-9-0/#new-resolver

Reply:
🤷‍♂️ you're the boss boss